pkg/selector/emr.go

```diff
@@ -0,0 +1,342 @@
+package selector
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+// EMR is a Service type for a custom service filter transform
+type EMR struct{}
+
+// Filters implements the Service interface contract for EMR
+func (e EMR) Filters(version string) (Filters, error) {
+	filters := Filters{}
+	instanceTypes, err := e.getEMRInstanceTypes(version)
+	if err != nil {
+		return filters, err
+	}
+	filters.InstanceTypes = &instanceTypes
+	filters.RootDeviceType = aws.String("ebs")
+	filters.VirtualizationType = aws.String("hvm")
+	return filters, nil
+}
+
+// getEMRInstanceTypes returns a list of instance types that emr supports
+func (e EMR) getEMRInstanceTypes(version string) ([]string, error) {
+	instanceTypes := []string{}
+	if version == "" {
+		version = "5.20.0"
```

Comment:
np: `const`

also is this latest, fallback, or both?

Reply:
both, we'll fallback to the latest.  👍  on const, I'll make that change

Reply:
I realized this is not the latest... I think this is the most utilized, I'll make const reflect that it's a fallback and not latest.